examples/with-claims/api/claims/RolesClaim.ts

```diff
@@ -0,0 +1,91 @@
+import Session, { SessionClaim, SessionClaimValidator, SessionContainer } from "supertokens-node/recipe/session";
```

Comment:
This should be tightly couple with our user roles recipe. For example, the developer should not have to call `addRole` on this claim AND in the session recipe.

Also, this is missing concept of permissions

Reply:
It does, but I'm not sure it's worth including in this example. When we provide a full claims implementation in the roles recipe it will export those claims instead of the user (or this example) implementing them.

Reply:
Im going to keep this open so that we don't forget about supporting permissions.

Reply:
By default, other auth providers add the user's roles and permissions into the JWT.